src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -1702,17 +1703,6 @@ tile.boiler_casing.steel_firebox.name=Steel Firebox Casing
 tile.boiler_casing.titanium_firebox.name=Titanium Firebox Casing
 tile.boiler_casing.tungstensteel_firebox.name=Tungstensteel Firebox Casing
 
-# Metal casing blocks
```

Comment:
Do you understand that names here are just DIFFERENT from the names you implemented with autogenerated casings? Have you ever thought about retaining old names for currently registered casings?

Reply:
The original name doest exist anymore and it's autogenerated with the name of the metal

Reply:
And look worse than original ones, nobody asked you to get rid of them. Bring old names back for casings currently available in gtce.

Reply:
name revert